README.md

```diff
@@ -33,15 +33,18 @@ To make Evervault available for use in your app:
 ```python
 import evervault
 
-# Initialize the client with your team’s API key
-evervault.init("<YOUR_API_KEY>")
+# Initialize the client with your App’s API key and App's UUID
```

Comment:
The public facing name is App ID. We should make sure the docs & errors reflect that.